### ISSUE TITLE
Missing accuracy scores in Table 1/2 of Paper

### ISSUE THREAD

**Issue by timothylimyl** [stanfordnlp/dspy#190]
Hi, may I know what happen to the missing results of the test set of Table 1/2 in the paper? 

As quoted from the paper in regards to Table 1:
```
We sample 200 and 300 question–answer pairs from the official training set for training and develop-
ment, respectively. Our final evaluations use the 1.3k official test set examples.
```
However, the evaluation for the official test set seem to be missing for a few examples.

**Comment by okhat** [comment]
Thanks for opening this issue. We report test set results only for representative methods. We don’t want to encourage overfitting on the test set by reporting every possible combination on it. The paper discusses this consideration directly.

**Comment by timothylimyl** [comment]
@okhat , thank you for the reply.

I do not quite understand what you mean by not wanting to encourage over-fitting on the test set by reporting every possible combination. I thought you only use the input-output pairs from the train+dev set to formulate the prompts?

**Comment by okhat** [comment]
Yes only train and dev dictate the prompts.

However, it is bad experimental protocol to evaluate on the test set more than is absolutely strictly necessary. Unfortunately this important condition is all but completely lost in many modern ML papers, but we think test sets should be as preserved as possible and not over-used.
